### PR TITLE
[SEM-069] Development of the 404, 500 Error Component 

### PR DESCRIPTION
**Pull request by Fernando-MV7**

### CODE REVIEW

File: src/ui/components/atoms/error-message/styles.scss [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+.error-message {
+  color: $ads-primary-main;
+  font-family: Arial, sans-serif;
+  padding: 10rem;
+  text-align: center;
+
+  h1 {
+    color: $ads-primary-main;
+    font-family: Inter, Arial, sans-serif;
```
Comment (Ditmar) [inline src/ui/components/atoms/error-message/styles.scss:12]: please review the figma I thing the font is Lato here

File: src/ui/components/atoms/error-message/styles.scss [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    font-size: 6rem;
+    font-weight: 400;
```
Comment (Ditmar) [inline src/ui/components/atoms/error-message/styles.scss:14]: please put this variable in the globals

File: src/ui/components/atoms/error-message/styles.scss [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+    @include respond-to('extra-large') {
+      font-size: 2.25rem;
+    }
+  }
+
+  .message {
+    color: $ads-primary-main;
+    font-size: 0.75rem;
+    font-weight: 400;
```
Comment (Ditmar) [inline src/ui/components/atoms/error-message/styles.scss:64]: The same here

File: src/ui/components/atoms/error-message/styles.scss [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+.server-error {
+  h1 { 
+    font-family: Inter, Arial, sans-serif;
+    font-size: 8rem;  
+    font-weight: 800;
+    line-height: 9.682rem;
+  }
+
+  .title { 
+    font-family: Inter, Arial, sans-serif;
```
Comment (Ditmar) [inline src/ui/components/atoms/error-message/styles.scss:119]: Check the font here

File: src/ui/components/atoms/error-message/styles.scss [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+    font-size: 2.5rem; 
+    font-weight: 700;
+    line-height: 3.026rem;
+  }
+
+  .message { 
+    font-family: Inter, Arial, sans-serif;
```
Comment (Ditmar) [inline src/ui/components/atoms/error-message/styles.scss:126]: The same here

**Review by Ditmar — CHANGES_REQUESTED** [review]
Nice Pr! good job please review the comments

File: src/ui/components/atoms/error-message/styles.scss (hunk 1 above)
Comment (Fernando-MV7) [reply src/ui/components/atoms/error-message/styles.scss:12]: Captura de pantalla (740)
the font on the figma is Inter and not Lato

**Pull request closed without merging**